makefile

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: GPL-2.0
+
+# Default target executed when no arguments are given to make.
+default_target: all
+
+.PHONY : default_target
+
+# Allow only one "make -f <foo>" at a time, but pass parallelism.
+.NOTPARALLEL:
+
+
+mobile:
+	cd build-mobile; LIBRARY_PATH=../install_root/lib make 
```

Comment:
this will fail until the first time build.sh was run and created that directory

Reply:
correct there should be a test for that.